ToDo.DataAccess/ToDo.DataAccess/ToDoRepository.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ToDo.DataAccess.DataBase;
+using ToDo.DataAccess.Models;
+
+namespace ToDo.DataAccess
+{
+    public class ToDoRepository : IToDoRepository
```

Comment:
The repository should be created for every table or group of tables. Not for all application. So, please, create it for categories, items and tags.

Reply:
But in this case I would recommend to create `GenericRepository` for CRUD operations.

Reply:
Changed.